### DISCUSSION TITLE
How to change order notes text?

### DISCUSSION THREAD

**Discussion by orschiro** [woocommerce/woocommerce#53965] [Checkout Flow]
I would like to change this following text. 

How can I?

![image](https://github.com/user-attachments/assets/fc783be1-649e-4360-acaf-46c347486159)

**Comment by senadir** [comment]
For now, you will need to use a translation plugin like `Say What` to update that. 

<img width="1160" alt="image" src="https://github.com/user-attachments/assets/1bff7139-d69b-4d1f-8117-b7f82e5522e2" />

**Reply by orschiro** [reply]
That's working, thanks! :)
